k8s/client.go

```diff
@@ -0,0 +1,288 @@
+	stork.SetInstance(stork.New(client, storkClient, snapClient))
+}
+
+// SetBaseClient sets the kubernetes clientset
+func (k *k8sOps) SetBaseClient(client kubernetes.Interface) {
+	admissionregistration.SetInstance(admissionregistration.New(client.AdmissionregistrationV1beta1()))
+	apps.SetInstance(apps.New(client.AppsV1(), client.CoreV1()))
+	batch.SetInstance(batch.New(client.BatchV1()))
+	core.SetInstance(core.New(client.CoreV1(), client.StorageV1()))
+	discovery.SetInstance(discovery.New(client.Discovery()))
+	rbac.SetInstance(rbac.New(client.RbacV1()))
+	storage.SetInstance(storage.New(client.StorageV1(), client.StorageV1beta1()))
+}
+
+// SetSnapshotClient sets the snapshot clientset
+func (k *k8sOps) SetSnapshotClient(snapClient rest.Interface) {
+	externalstorage.SetInstance(externalstorage.New(snapClient))
+}
+
+// SetStorkClient sets the stork clientset
+func (k *k8sOps) SetStorkClient(storkClient storkclientset.Interface) {
+}
+
+// SetOpenstorageOperatorClient sets the openstorage operator clientset
+func (k *k8sOps) SetOpenstorageOperatorClient(ostClient ostclientset.Interface) {
+	operator.SetInstance(operator.New(ostClient))
+}
+
+// SetAPIExtensionsClient sets the api extensions clientset
+func (k *k8sOps) SetAPIExtensionsClient(apiExtensionsClient apiextensionsclient.Interface) {
+	apiextensions.SetInstance(apiextensions.New(apiExtensionsClient))
+}
+
+// SetDynamicClient sets the dynamic clientset
+func (k *k8sOps) SetDynamicClient(dynamicClient dynamicclient.Interface) {
+	dynamic.SetInstance(dynamic.New(dynamicClient))
+}
+
+// SetOpenshiftAppsClient sets the openshift apps clientset
+func (k *k8sOps) SetOpenshiftAppsClient(ocpAppsClient ocp_clientset.Interface) {
+}
+
+// SetOpenshiftSecurityClient sets the openshift security clientset
+func (k *k8sOps) SetOpenshiftSecurityClient(ocpSecurityClient ocp_security_clientset.Interface) {
+}
+
+// SetAutopilotClient sets the autopilot clientset
+func (k *k8sOps) SetAutopilotClient(autopilotClient autopilotclientset.Interface) {
+	autopilot.SetInstance(autopilot.New(autopilotClient))
+}
+
+// SetTalismanClient sets the talisman clientset
+func (k *k8sOps) SetTalismanClient(talismanClient talismanclientset.Interface) {
+	talisman.SetInstance(talisman.New(talismanClient))
+}
+
+// SetPrometheusClient sets the prometheus clientset
+func (k *k8sOps) SetPrometheusClient(prometheusClient prometheusclient.Interface) {
+	prometheus.SetInstance(prometheus.New(prometheusClient))
+}
+
+func (k *k8sOps) loadClientFromKubeconfig(kubeconfig string) error {
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		return err
+	}
+
+	return k.loadClientFor(config)
+}
+
+func (k *k8sOps) loadClientFromConfigBytes(kubeconfig []byte) error {
+	config, err := clientcmd.RESTConfigFromKubeConfig(kubeconfig)
+	if err != nil {
+		return err
+	}
+
+	return k.loadClientFor(config)
+}
+
+func (k *k8sOps) loadClientFor(config *rest.Config) error {
```

Comment:
I think we will not need any of the loadClient methods. Check comments in `k8s/k8s.go` for more details.